ansible_builder/cli.py

```diff
@@ -25,17 +25,17 @@ def run():
             print(e.args[0])
             sys.exit(1)
     elif args.action == 'introspect':
-        for folder in args.folders:
-            data = process(folder)
-            if args.sanitize:
-                data['python'] = sanitize_requirements(data['python'])
-                print()
-                print('Sanitized dependencies for {0}'.format(folder))
-                print(yaml.dump(data, default_flow_style=False))
-            else:
-                print()
-                print('Dependency data for {0}'.format(folder))
-                print(yaml.dump(data, default_flow_style=False))
+        data = process(args.folder, user_pip=args.user_pip, user_bindep=args.user_bindep)
```

Comment:
Changed `args.folders` to `args.folder` - because the original idea was that we would introspect all the default Ansible collection search paths. This is unlikely to ever be supported. It's also very hard to reconcile with combining with the user-provided files.

That's a major change - before, we combined the user's content with the collection content _after_ the introspect script. Now, I am moving the user's files into the image and having the introspect script include that content in its output.